metadata.json

```diff
@@ -84,6 +84,6 @@
     }
   ],
   "pdk-version": "1.10.0",
-  "template-url": "https://github.com/puppetlabs/pdk-templates#master",
-  "template-ref": "heads/master-0-gf778803"
+  "template-url": "https://github.com/puppetlabs/pdk-templates/#1.10.0",
+  "template-ref": "1.10.0-0-gbba9ac3"
```

Comment:
The pdk update has reverted the template to the template bundled with pdk 1.10.0.